jobs/JGLOBAL_ATMOS_VMINMON

```diff
@@ -1,42 +1,25 @@
 #! /usr/bin/env bash
 
 ###########################################################
-# GDAS Minimization Monitor (MinMon) job
+# Global Minimization Monitor (MinMon) job
 ###########################################################
 source "${HOMEgfs}/ush/preamble.sh"
 source "${HOMEgfs}/ush/jjob_header.sh" -e "vminmon" -c "base vminmon"
 
-###########################################################
-# obtain unique process id (pid) and make temp directories
-###########################################################
-export MINMON_SUFFIX=${MINMON_SUFFIX:-${NET}}
-export m_job=${m_job:-${MINMON_SUFFIX}_mmDE}
-
-
 ##############################################
 # Specify Package Areas
 ##############################################
-export HOMEgfs=${HOMEgfs:-${NWROOT}/gfs.${gfs_ver}}
+export M_FIX=${M_FIX:-${HOMEgfs}/fix/mon}
 export SCRgfs=${SCRgfs:-${HOMEgfs}/scripts}
-
-export M_FIXgdas=${M_FIXgdas:-${HOMEgfs}/fix/gdas}
-
-export HOMEminmon=${HOMEminmon:-${HOMEgfs}}
-export EXECminmon=${EXECminmon:-${HOMEminmon}/exec}
-export USHminmon=${USHminmon:-${HOMEminmon}/ush}
-
+export USHminmon=${USHminmon:-${HOMEgfs}/ush}
 
 #############################################
-#  determine PDY and cyc for previous cycle
+# Determine PDY and cyc for previous cycle
```

Comment:
The convention is `GDATE` for previous cycle.  Is there a reason why GSI monitoring is defining its own names?  
Should use `${assim_freq}` instead of `-6` below.

Reply:
Have updated to use `GDATE`, `gPDY`, `gcyc`, and `assim_freq`.